common/models/user.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "user",
   "base": "PersistedModel",
+  "strict": "throw",
+  "persisteUndefinedAsNull": true,
   "trackChanges": true,
   "properties": {
     "id": {
       "id": true,
-      "type": "string"
+      "type": "string",
+      "defaultFn": "guid"
```

Comment:
The User model does not support replication/sync ATM. Please revert these changes.